cygnus-arcgis/src/main/java/com/telefonica/iot/cygnus/backends/arcgis/baselogger/BaseLogger.java

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
All new files should include the standard Cygnus license header

(General comment, it applies to all the new files addd in this PR)

Reply:
Done c8d4419